### ISSUE TITLE
Feature request: add ETH assets in SVG

### ISSUE THREAD

**Issue by konopkja** [ethereum/ethereum-org-website#8026]
### Is your feature request related to a problem? Please describe.

There are many reasons why someone might need our assets in SVG (especially the glyph logo section)

Current issues:

1. Our website does not provide svg format on many of the brand assets
2. If there is SVG version on the website, there is no download button

### Describe the solution you'd like

1. provide an option to download ethereum glyph in svg format
2. add download button for svg assets

### Describe alternatives you've considered

some users also requested EPS format, but SVG would be enough.

**Comment by konopkja** [comment]
This has been requested via the feedback widget on ethereum.org multiple times

**Comment by minimalsm** [comment]
> some users also requested EPS format, but SVG would be enough.

To add to this: IMO, I don't think we would want to add EPS format downloads, even if we had the files. It's an obscure file format that would distract more users from accomplishing their goals than it would help.

**Comment by ankitzm** [comment]
Adding a downloadable SVG makes a lot of sense here.
Would love to take up this issue !!

**Comment by abhinow01** [comment]
hey, I'm new to contributing to open source and I would love to work on this issue !

**Comment by konopkja** [comment]
Hi, @ankitzm was first, im assigning you this task. Still willing to contribute?

**Comment by abhinow01** [comment]
Yes, I will be more than happy to contribute

**Comment by ankitzm** [comment]
hey @konopkja 
I think, SVG asset of Ethereum can be added here - https://ethereum.org/en/assets/#brand
Do you have the SVG versions of the assets that you prefer adding here, would add them all.

**Comment by konopkja** [comment]
> hey @konopkja I think, SVG asset of Ethereum can be added here - https://ethereum.org/en/assets/#brand Do you have the SVG versions of the assets that you prefer adding here, would add them all.

Correct, the assets should be on the asset page. We do not have SVG versions, where there is none.

**Comment by kaspi90** [comment]
Hi, I would like to work on this issue. Can you assign the task to me?

**Comment by kaspi90** [comment]
I fixed the issue and created a PR https://github.com/ethereum/ethereum-org-website/pull/8891

**Comment by konopkja** [comment]
Thank you so much @kaspi90 for your contribution! Looking good, maybe just one question: when i download and open the svgs - it shows 1x1 px size. Is it possi
<img width="215" alt="Screenshot 2022-12-15 at 17 17 41" src="https://user-images.githubusercontent.com/100724231/207857661-04818880-0c73-4db0-a703-1e0c9f266a60.png">
ble to have the default downloaded .svg bigger? Lets say 64x64?

**Comment by kaspi90** [comment]
I'll look at it and send you an update

**Comment by kaspi90** [comment]
Hi, I added the 64x64 size

**Comment by konopkja** [comment]
Thank you so much @kaspi90 ! I will let the team know for review and merge. This will likely happen after Christmas.
